pypeit/calibrations.py

```diff
@@ -303,7 +303,9 @@ def get_arc(self):
         self.msarc = self.arcImage.load()
         if self.msarc is None:  # Otherwise build it
             msgs.info("Preparing a master {0:s} frame".format(self.arcImage.frametype))
-            self.msarc = self.arcImage.build_image(bias=self.msbias, bpm=self.msbpm)
+            ## FW: turn off bias subtraction for Arc since XSHOOTER will only take arc with 1x1.
```

Comment:
I see now that this exists already (thought it did).

So you just need to do something like  

```
[calibrations][arcframe][process][bias] = "skip"
```

Reply:
We figured out how to address this and changed it back. Note that your preferred option does not appear to work because of what looks like a big bug in the pypeit file reader. About to submit an issue on that.